app/helpers/mailer_helper.rb

```diff
@@ -0,0 +1,12 @@
+module MailerHelper
+  def email_footer
+    uuid = Setting[:instance_id]
+    title = Setting[:instance_title]
+
+    if uuid.present? & title.present?
+      _('This email was sent from Foreman instance %{title} and with identifier %{uuid}') % {title: title, uuid: uuid}
+    elsif title.empty?
```

Comment:
Since we only care about uuid in this case, I'd check for its presence.
```suggestion
    elsif uuid.present?
```